boards/nxp/imx95_evk/board.c

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2024 NXP
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/devicetree.h>
+#include <zephyr/init.h>
+#include <zephyr/kernel.h>
+#include <soc.h>
+
+static int board_init(void)
+{
```

Comment:
Please look at the changes made as part of this PR: https://github.com/zephyrproject-rtos/zephyr/pull/85605 and update this implementation accordingly.

Reply:
Actually the soc/board early init hooks are not able to be used for the SoC based on SCMI APIs.
I had commented in file.

```
/*
 * Because platform is using ARM SCMI, drivers like scmi, mbox etc. are
 * initialized during PRE_KERNEL_1. Common init hooks is not able to use.
 * SoC early init and board early init could be run during PRE_KERNEL_2 instead.
 */
```